doc/administrator/web/index.md

```diff
@@ -175,9 +175,16 @@ The different application profiles and their functions are the following:
 
 To launch an IRIDA application server with one (or more) of these profiles, you must enable the profile with the `spring.profiles.active` variable in your Tomcat configuration.  For example to run an IRIDA server with the `web` and `analysis` profiles active, you would set the following configuration:
 
-`spring.profiles.active=web,analysis`
+```
+spring.profiles.active=web,analysis
+```
+
+Using the multi server configuration it's also suggested you use the `irida.db.profile` setting in Tomcat:
```

Comment:
Could you describe what `irida.db.profile` does?  E.g., will IRIDA break if it's not set?

Reply:
Done in e1663c0